modulefiles/modulefile.ww3.wcoss2.lua

```diff
@@ -0,0 +1,22 @@
+help([[
```

Comment:
We need to work w/ the WW3 developers and see how to do this better.
WW3 repo has a build system and we should see how they assign/handle dependencies.

Reply:
#TODO later.